readme.md

```diff
@@ -142,7 +142,7 @@ Thanks for contributing! 🦋🙌
 
 ### File management
 
-- [](# "download-folder-button") [Adds a button to a download button entire folders.](https://user-images.githubusercontent.com/1402241/35044451-fd3e2326-fbc2-11e7-82e1-61ec7bee612b.png) *Uses [download-directory.github.io](https://download-directory.github.io)*
+- [](# "download-folder-button") [Adds a button to a download entire folders.](https://user-images.githubusercontent.com/1402241/35044451-fd3e2326-fbc2-11e7-82e1-61ec7bee612b.png) *Uses [download-directory.github.io](https://download-directory.github.io)*
```

Comment:
Hmm, why not remove the redundant `a` as well

Reply:
yes